test/index.test.js

```diff
@@ -34,175 +33,153 @@ describe('module-resolver', () => {
       const rootTransformerOpts = {
         babelrc: false,
         plugins: [
-          [plugin, {
-            root: './test/testproject/src',
-          }],
+          [
+            plugin,
+            {
+              root: './test/testproject/src',
+            },
+          ],
         ],
       };
 
       it('should resolve the file path', () => {
-        testWithImport(
-          'app',
-          './test/testproject/src/app',
-          rootTransformerOpts,
-        );
+        testWithImport('app', './test/testproject/src/app', rootTransformerOpts);
       });
 
       it('should resolve the sub file path', () => {
         testWithImport(
           'components/Root',
           './test/testproject/src/components/Root',
-          rootTransformerOpts,
+          rootTransformerOpts
         );
       });
 
       it('should resolve a sub file path without /index', () => {
         testWithImport(
           'components/Header',
           './test/testproject/src/components/Header',
-          rootTransformerOpts,
+          rootTransformerOpts
         );
       });
 
       it('should resolve the file path while keeping the extension', () => {
         testWithImport(
           'components/Header/header.css',
           './test/testproject/src/components/Header/header.css',
-          rootTransformerOpts,
+          rootTransformerOpts
         );
       });
 
-      it('should resolve the file path with an extension that is non-standard in node', () => {
+      it('should resolve the file path while keeping the extension and query', () => {
```

Comment:
This file hasn't been prettified in 2 years so I was the lucky one to make that change, but here's my first unit test asserting that a path with a query string is resolved and transformed successfully for the root path case.